app/services/typicode.js

```diff
@@ -1,14 +1,22 @@
 const request = require('request-promise'),
   logger = require('../logger'),
   typicodePath = 'https://jsonplaceholder.typicode.com',
-  { apiError } = require('../errors');
+  { apiError } = require('../errors'),
+  lodash = require('lodash');
 
 const options = endpoint => ({
   uri: endpoint,
   json: true,
   resolveWithFullResponse: false
 });
 
+const mapPhotos = (album, photos) => {
+  album.photos = photos.map(photo => ({
```

Comment:
dont reassign parameter values, instead of mutating the original array why dont you just do it in a more functional manner?